lib/resources/rds/db_instance.rb

```diff
@@ -54,6 +56,21 @@ def with_encrypted_storage?
             @dbi.storage_encrypted
           end
 
+          # Tests whether the database should allow connections from the given CIDR range.
+          # Inspects each of the security groups associated with the instance and associated inbound rules
+          # to check for a rule matching the given CIDR range.
+          # For external connections, it is verified that the database is publicly accessible.
+          def accessible_from?(cidr)
```

Comment:
Given the description and caveats, is this an appropriate name?

It's effectively testing the security rules do not prevent connection - not that you would actually be able to connect.